wms/src/components/ui/workflow-tracker.tsx

```
@@ -0,0 +1,235 @@
+
+function CheckIcon({ className }: { className?: string }) {
+  return (
+    <svg
+      className={className}
+      viewBox="0 0 16 16"
+      fill="none"
+      stroke="currentColor"
+      strokeWidth={2.5}
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    >
```

Biome (2.5.0)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)